docs/src/main/asciidoc/datasource.adoc

```diff
@@ -511,154 +504,155 @@ public class TestResources {
 
 The Microsoft SQL Server JDBC driver works essentially the same as the others.
-More details can be found in the https://docs.microsoft.com/en-us/sql/connect/jdbc/connecting-to-sql-server-with-the-jdbc-driver?view=sql-server-2017[official documentation].
+More details can be found in the link:https://docs.microsoft.com/en-us/sql/connect/jdbc/connecting-to-sql-server-with-the-jdbc-driver?view=sql-server-2017[official documentation].
 
-=== MySQL
+==== MySQL
 
 `jdbc:mysql:[replication:|failover:|sequential:|aurora:]//<hostDescription>[,<hostDescription>...]/[database][?<key1>=<value1>[&<key2>=<value2>]]`
-hostDescription:: `<host>[:<portnumber>]  or address=(host=<host>)[(port=<portnumber>)][(type=(master|slave))]`
+hostDescription:: `<host>[:<portnumber>] or address=(host=<host>)[(port=<portnumber>)][(type=(master|slave))]`
 
 Example:: `jdbc:mysql://localhost:3306/test`
 
-You can find more information about this feature and others detailed in the https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference.html[official documentation].
+More information about this feature and others is detailed in the link:https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference.html[official documentation].
+
+===== MySQL Limitations
 
-==== MySQL Limitations
+When compiling a Quarkus application to a native image, the MySQL support for JMX and Oracle Cloud Infrastructure (OCI) integrations are disabled as they are incompatible with GraalVM native images.
 
-When compiling a Quarkus application to native-image, the MySQL support for JMX and Oracle Cloud Infrastructure (OCI) integrations are disabled as they are not compatible
-with GraalVM native-images.
-The lack of JMX support is a natural consequence of running in native and is unlikely to be resolved.
-The integration with OCI could be resolved, if you need it we suggest opening a support request with the MySQL Connector/J maintainers.
+* The lack of JMX support is a natural consequence of running in native mode and is unlikely to be resolved.
+* The integration with OCI is not supported.
 
-=== Oracle
+==== Oracle
 
 `jdbc:oracle:driver_type:@database_specifier`
 
 Example:: `jdbc:oracle:thin:@localhost:1521/ORCL_SVC`
 
-More details can be found in the https://docs.oracle.com/en/database/oracle/oracle-database/21/jjdbc/data-sources-and-URLs.html#GUID-AEA8E228-1B21-4111-AF4C-B1F33744CA08[official documentation].
+More details can be found in the link:https://docs.oracle.com/en/database/oracle/oracle-database/21/jjdbc/data-sources-and-URLs.html#GUID-AEA8E228-1B21-4111-AF4C-B1F33744CA08[official documentation].
 
-=== PostgreSQL
+==== PostgreSQL
 
-PostgreSQL only runs as a server, as do the rest of the databases below.
-As such, you must specify connection details, or use the defaults.
+PostgreSQL and the rest of the databases mentioned below run only as a server.
+As such, specify connection details or use the defaults.
```

Comment:
I'm not sure what this means. Maybe it's meant as opposed to H2 that can run in embedded mode? If so, I'd remove this sentence: it's confusing, unnecessary and not in the right place since some of the DBs above (Oracle, MySQL, ...) also cannot be embedded.

Reply:
I am not the author of the original text, but by applying what you shared here:

As opposed to H2 that can run in embedded mode, PostgreSQL and the rest of the databases mentioned below run only as a server.
As such, specify connection details or use the defaults.


Would that work, or are you for dropping lines 610 and 611 completely? 
@yrodiere

Reply:
As stated above it's not just the content, it's also just incorrect as there are also DBs mentioned *above* that only run as a server.

Making it right would be more work than it's worth, so let's just drop it?

Reply:
OFC, will do. One sec.